src/families/hedera/js-getTransactionStatus.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  AmountRequired,
+  NotEnoughBalance,
+  InvalidAddress,
+  InvalidAddressBecauseDestinationIsAlsoSource,
+  RecipientRequired,
+} from "@ledgerhq/errors";
+import * as hedera from "@hashgraph/sdk";
+import type { Transaction } from "./types";
+import type { Account, TransactionStatus } from "../../types";
+import { calculateAmount } from "./utils";
+
+export default async function getTransactionStatus(
+  account: Account,
+  transaction: Transaction
+): Promise<TransactionStatus> {
+  const errors: Record<string, Error> = {};
+
+  if (!transaction.recipient || transaction.recipient.length === 0) {
+    errors.recipient = new RecipientRequired("");
+  } else {
+    const senderAccountId = hedera.AccountId.fromString(account.seedIdentifier);
+
+    try {
+      const recipientAccountId = hedera.AccountId.fromString(
+        transaction.recipient
+      );
+
+      if (senderAccountId.equals(recipientAccountId)) {
+        errors.recipient = new InvalidAddressBecauseDestinationIsAlsoSource("");
```

Comment:
Is it forbidden to self-send with Hedera?

Reply:
To the best of my knowledge, yes it is forbidden to self-send.